sentences/hr/cover_HassTurnOff.yaml

```diff
@@ -2,52 +2,91 @@ language: hr
 intents:
   HassTurnOff:
     data:
+      #windows
       - sentences:
-          - <zatvori> {name} (prozor(e)) [u|na] {area}
           - <zatvori> [sve] prozor[e] [u|na] {area}
           - <zatvori> {area} prozor[e]
         slots:
-          device_class:
-            - window
+          device_class: window
           domain: cover
         response: windows_area
 
       - sentences:
-          - <zatvori> {name} (vrata) [u|na] {area}
-          - <zatvori> [sva] vrata [u|na] {area}
-          - <zatvori> {area} vrata
-        slots:
-          device_class:
-            - door
+          - <zatvori> <name>
+        requires_context:
           domain: cover
-        response: doors_area
+        slots:
+          device_class: window
+        response: window
 
       - sentences:
-          - <zatvori> {name} [zavjes[e|u]|zastor(e)] [u|na] {area}
-        response: curtains_area
-        slots:
-          device_class:
-            - curtain
+          - <zatvori> <name> [u|na] {area}
+        requires_context:
           domain: cover
+        slots:
+          device_class: window
+        response: window
 
+      #door
       - sentences:
-          - <zatvori> {name} [rolet[e|u]] [u|na] {area}
-        response: blinds_area
+          - <zatvori> [sva] vrata [u|na] {area}
+          - <zatvori> {area} vrata
         slots:
-          device_class:
-            - blind
+          device_class: door
           domain: cover
+        response: door
 
+      #error: Expected window, got door for slot device_class for: zatvori ulazna vrata
```

Comment:
@synesthesiam can you please help me with this error. Test sentence is "Close the main door", but it ignores device class.

Reply:
It's getting caught by this template:

```yaml
      - sentences:
          - <zatvori> <name>
        requires_context:
          domain: cover
        slots:
          device_class: window
        response: window
```

because the name of the entity is "ulazna vrata"

Reply:
I am aware of that. But, in fixtures "ulazna vrata" has device class "door", not "window". I am trying to write different responses for windows and doors.

Reply:
Never mind, I will rewrite it with device name instead of class.